src/Etg.Yams.Core/Process/Process.cs

```diff
@@ -61,9 +61,7 @@ public Process(string exePath, bool showProcessWindow)
 
                 if (!_showProcessWindow)
                 {
-                    _process.StartInfo.RedirectStandardOutput = true;
                     _process.StartInfo.UseShellExecute = false;
-                    _process.StartInfo.CreateNoWindow = true;
```

Comment:
When you run the tests, are console windows popping up? The goal of this was to hide the console when the showProcessWindow setting is false.

Reply:
I tried it and it seems to be working as expected